packages/squiggle-lang/src/rescript/Utility/E.res

```diff
@@ -1,4 +1,3 @@
-open Rationale.Function.Infix
 module FloatFloatMap = {
```

Comment:
Leave a comment on the top of this file saying that several functions were taken from Rationale, and link to the github file. (Ideally giving a summary of what functions were taken)